app/src/main/java/eu/zimbelstern/tournant/ui/MainActivity.kt

```diff
@@ -256,6 +257,18 @@ class MainActivity : AppCompatActivity(), RecipeListAdapter.RecipeListInterface
 			intent.action = ""
 		}
 
+		lifecycleScope.launch { // BACKUP RECIPES
+			viewModel.idsRecipesFiltered.collectLatest {
```

Comment:
You are triggering a backup on each change in the displayed recipes here. So on each search query change etc.